cumulusci/conftest.py

```diff
@@ -108,3 +108,9 @@ def open_db():
                 yield connection, Base.metadata, session
 
         yield open_db
+
+
+@pytest.fixture(scope="session", autouse=True)
+def disable_exit(request):
+    with mock.patch("sys.exit"):
+        yield
```

Comment:
This doesn't strike me as a good idea, since it could cause us to overlook code that is exiting when it shouldn't. What's the rationale for patching it everywhere instead of a targeted patch or exception handler?

Reply:
Okay, removed. I was trying to reduce the mocking burden as you described. But I accept your criticisms of this particular approach.

Reply:
I think maybe the way to go is to create a context manager or pytest fixture (`cli_test_harness`?) that sets up all the existing mocks. That way at least they are centralized and can be requested with a single line instead of a giant stack of patches.

Reply:
Not saying that needs to be in the PR though. I'll approve.